araig_test_runners/scripts/test_1_braking

```diff
@@ -121,6 +121,7 @@ class Test1(TestBase):
             self._publishers["test_succeeded"].publish(self.buildNewBoolStamped(False))
             self._publishers["test_completed"].publish(self.buildNewBoolStamped(False))
             # !!! ui should set reset_test to False after set to True, reset signal is an event
+            rospy.sleep(2)
```

Comment:
Remove. TODO: This will be eventually fixed by https://github.com/ipa320/araig_test_stack/issues/39